Tests/GRPCTests/GRPCKeepaliveTests.swift

```diff
@@ -0,0 +1,122 @@
+/*
+ * Copyright 2020, gRPC Authors All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+@testable import GRPC
+import NIO
+import EchoModel
+import EchoImplementation
+import XCTest
+
+class GRPCKeepaliveTests: GRPCTestCase {
```

Comment:
Could you run `swift test --generate-linuxmain` to generate the test manifest for Linux?